### ISSUE TITLE
fd low latency master todo list

### ISSUE THREAD

**Issue by pgkeller** [tenstorrent/tt-metal#11796]
- [] #11789 
 - [] #11307 
 - [] #11790 
 - [] #11791 
 - [] #11792
 - [] #11895
 - [] #11794 
 - [] #11844 
 - [] (deprioritized, can handle later) #12979
 - [] #11795

**Comment by pgkeller** [comment]
added #11307 to master list: plan is to deprecate l1_data section entirely

**Comment by pgkeller** [comment]
Current status: using the ring buffer for kernel bins was reverted due to a non-deterministic hang in unet, that is being debugged but timeline is hard to estimate, eta early next week ~11/5.  After that the transfers need to be done async, roughly 3 days, etc end of next week, ~11/8.

**Comment by jbaumanTT** [comment]
I broke out the unet hang to #14501

**Comment by razorback3** [comment]
Is this issue fully completed?

**Comment by razorback3** [comment]
Is this all done?
Can we close this issue as all completed?

**Comment by pgkeller** [comment]
yes, there is more work happening as follow up work, but this has been re-enabled.  closed.
